mne/time_frequency/tfr.py

```diff
@@ -245,6 +283,239 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
         yield tfr
 
 
+# Loop of convolution: single trial
+
+
+def timefreq_transform(epoch_data, frequencies, sfreq=1.0, method='morlet',
+                       n_cycles=7.0, zero_mean=None, time_bandwidth=4.0,
+                       use_fft=True, decim=1, output='complex', n_jobs=1,
+                       verbose=None):
+    """Computes time frequency transforms.
+
+    Parameters
+    ----------
+    epoch_data : array of shape (n_epochs, n_channels, n_times)
+        The epochs.
+    frequencies : array-like of floats, shape (n_freqs)
+        The frequencies.
+    sfreq : float | int, defaults to 1.0
+        Sampling frequency of the data.
+    method : 'mtm' | 'morlet', defaults to 'morlet'
+        The time frequency method. 'morlet' convolves a Morlet wavelet. 'mtm'
+        convolves DPSS tapers.
+    n_cycles : float | array of float, defaults to 7.0
+        Number of cycles  in the Morlet wavelet. Fixed number
+        or one per frequency.
+    zero_mean : bool
```

Comment:
... defaults to None, which will use Tre for method='mtm' and False for method='morlet'.